### DISCUSSION TITLE
When saving to csv format, add an option not to save the response.

### DISCUSSION THREAD

**Discussion by TomTom18-eng** [projectdiscovery/httpx#1168] [Ideas]
When saving to csv format, add an option not to save the response. Because saving the response makes the file too large！！！！

**Comment by MetzinAround** [comment]
Hi @TomTom18-eng, thanks for the idea. I'm changing the category to ideas to better reflect the content of this discussion.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
